### PR TITLE
ps-5.6-5831

### PR DESCRIPTION
**Pull request by patrickbirch**
Updated the release notes with issues based on the release list.

### CODE REVIEW

File: doc/source/release-notes/Percona-Server-5.6.45-87.1.rst [hunk 1]
```diff
@@ -0,0 +1,51 @@
+.. rn:: 5.6.45-87.1
```
Comment (hrvojem) [inline doc/source/release-notes/Percona-Server-5.6.45-87.1.rst]: `5.6.45-86.1`

File: doc/source/release-notes/Percona-Server-5.6.45-87.1.rst [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+
+Find the release notes for Percona Server for MySQL 5.6.45-87.1 in
+`online documentation
+<https://www.percona.com/doc/percona-server/5.6/index.html>`_. Report
+bugs in the `Jira bug tracker
+<https://jira.percona.com/projects/PS>`_.
+
+
+.. |release| replace:: 5.6.45-87.1
```
Comment (hrvojem) [inline doc/source/release-notes/Percona-Server-5.6.45-87.1.rst]: `5.6.45-86.1`

File: doc/source/release-notes/Percona-Server-5.6.45-87.1.rst [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+- :psbug:`5752`
+- :psbug:`5780`
+- :psbug:`5833`
+- :psbug:`5725`
+- :psbug:`5742`
+- :psbug:`5743`
+- :psbug:`5746`
+
+
+Find the release notes for Percona Server for MySQL 5.6.45-87.1 in
```
Comment (hrvojem) [inline doc/source/release-notes/Percona-Server-5.6.45-87.1.rst]: `5.6.45-86.1`

**Comment by hrvojem** [comment]
version in the `con.py` file needs to be updated as well.

**Review by hrvojem — CHANGES_REQUESTED** [review]

**Comment by patrickbirch** [comment]
Updated the version number in the release notes, updated the version number in conf.py and added the release notes to the release notes index.

File: doc/source/release-notes/Percona-Server-5.6.45-86.1.rst [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+
+This release merges changes of `MySQL 5.6.45
+<https://dev.mysql.com/doc/relnotes/mysql/5.6/en/news-5-6-45.html>`_, including
+all the bug fixes in it. Percona Server for MySQL 5.6.45-86.1 is now the current
+GA release in the 5.6 series. All of Percona’s software is open-source and free.
+
+Bugs Fixed
+================================================================================
+
+- An option has been added to conditionally enable or disable hot backup library trace information dumped to stdout. Bug fixed :psbug:`4850`
```
Comment (george-lorch) [inline doc/source/release-notes/Percona-Server-5.6.45-86.1.rst]: Not correct.  Please use the same description used in the 8.0 release notes for the same issue:
The TokuDB hot backup library continually dumps TRACE information to the server error log. The user cannot enable or disable the dump of this information. Bug fixed #4850.

**Review by george-lorch — CHANGES_REQUESTED** [review]

File: doc/source/release-notes/Percona-Server-5.6.45-86.1.rst (hunk 4 above)
Comment (patrickbirch) [reply doc/source/release-notes/Percona-Server-5.6.45-86.1.rst]: updated.

**Review by hrvojem — APPROVED** [review]

**Pull request merged**